src/sidenav/sidenav-item.styles.tsx

```diff
@@ -42,10 +47,12 @@ export const DefaultButton = styled(Button.Default)<StyleProps>`
         span {
             div {
                 background-color: ${Color.Accent.Light[5]};
+                color: ${Color.Primary} !important;
             }
 
             span {
                 font-weight: 600 !important;
```

Comment:
there's a visual bug in safari. replace use of `font-weight` with the appropriate `TextStyleHelper`.
<img width="116" alt="Screenshot 2023-07-03 at 8 46 15 AM" src="https://github.com/LifeSG/react-design-system/assets/16853062/7f58ea22-89f9-4975-bcfd-5c3bfb6a9ffd">